pkg/restapi/v1/oidc4ci/controller.go

```diff
@@ -652,8 +652,6 @@ func (c *Controller) OidcAcknowledgement(e echo.Context) error {
 
 	var finalErr error
 
-	// todo: according to the spec those 2 fields are required, so condition is redundant.
-	//  Should be removed during endpoint update to the latest spec.
 	if body.NotificationId != nil && body.Event != nil {
```

Comment:
I think you can remove this condition as well https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0-ID1.html#section-10.1-2.1